src/tests/suite/daos_test_common.c

```diff
@@ -628,8 +628,8 @@ d_rank_t ranks_to_kill[MAX_KILLS];
 bool
 test_runable(test_arg_t *arg, unsigned int required_nodes)
 {
-	int		 i;
-	static bool	 runable = true;
+	int	i;
+	int	runable = 1;
```

Comment:
was landed in a different patch IIRC and not relevant to this one.

Reply:
Right. I accidentally included src/tests/suite/daos_test_common.c in one commit of this PR. I intended to use the current latest daos_test_common.c to replace the one I modified. I need to revert my change of daos_test_common.c which is a part of my first commit in this PR. Shall I use "git revert"? Thank you!